tox.ini

```diff
@@ -1,20 +1,31 @@
+[gh-actions]
```

Comment:
This works for sure, as it runs in psycopg test suite:
```
[tox]
envlist = {3.6,3.7,3.8,3.9,3.10}
isolated_build = True

[testenv]
commands =
    pytest {posargs}
extras = test
```

`gh-actions` is not necessary.